html/signup.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓 - 로그인</title>
+  <meta name="description" content="판다와 함께 더 나은 서비스와 향상된 쇼핑생활을 만끽하세요" />
+  <meta property="og:image" content="../resources/images/img_home_bottom.png" />
+  <meta property="og:title" content="" />
+  <meta property="og:description" content="" />
+  <meta property="og:url" content="" />
+  <meta property="og:type" content="website" />
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:image" content="../resources/images/img_home_bottom.png" />
+  <meta name="twitter:title" content=" " />
+  <meta name="twitter:description" content="" />
+  <link rel="shortcut icon" href="../resources/images/common/favicon.ico" type="image/x-icon" />
+  <link rel="icon" href="../resources/images/common/favicon.ico" type="image/x-icon" />
+  <link rel="preload" href="../resources/css/main.css" as="style" onload="this.onload=null;this.rel='stylesheet'" />
+</head>
+<body>
+  <div class='wrap'>
+    <div class='login_container'>
+      <h2 class='log_tit' tabindex="0"><img src="../resources/images/common/logo_tit.png" alt="판다마켓"></h2>
```

Comment:
❗️ 수정요청
현 태그가 상호작용이 불가능한데 tabindex 속성을 통해 focus가 되게 할 필요가 없을 것 같아요~
요구사항 상에서 ""판다마켓" 클릭 시 루트 페이지("/")로 이동합니다.(새로고침)"를 위해 a 태그로 변경하시는 것을 추천드려요!

```suggestion
      <h1 class='log_tit'> // 로고에는 일반적으로 h1 태그를 사용합니다
        <a href="/">
          <img src="../resources/images/common/logo_tit.png" alt="판다마켓">
        </a>
      </h1>
```